tests/sav/base_adaptive_full_alt2.res

```diff
@@ -1,16 +1,7 @@
-alt/base_adaptive_full_alt2.nit:55,10: Type Error: expected `B`, got `C`.
-alt/base_adaptive_full_alt2.nit:71,10: Type Error: expected `B`, got `C`.
-alt/base_adaptive_full_alt2.nit:84,11: Type Error: expected `B`, got `C`.
-alt/base_adaptive_full_alt2.nit:91,10: Type Error: expected `A`, got `Object`.
-alt/base_adaptive_full_alt2.nit:95,10: Type Error: expected `B`, got `Object`.
```

Comment:
To improve the proof of great success, uncomment the successful `#alt2#` from `base_adaptive_full.nit`.
You should also update the module comment.